### ISSUE TITLE
sup-config crash on macosx

### ISSUE THREAD

**Issue by prudnitskiy** [sup-heliotrope/sup#546]
sup-config immediately crashing right after start on macos 10.12

checked on system ruby (2.0.0), ruby 2.2 and ruby 2.5 (stable latest).

error stack:
```
> sup-config
[DEPRECATION] This gem has been renamed to optimist and will no longer be supported. Please switch to optimist as soon as possible.
/Users/logan/.rvm/gems/ruby-2.2.10/gems/sup-0.22.1/bin/sup-config:120:in `<top (required)>': undefined method `wrap_at=' for nil:NilClass (NoMethodError)
	from /Users/logan/.rvm/gems/ruby-2.2.10/bin/sup-config:23:in `load'
	from /Users/logan/.rvm/gems/ruby-2.2.10/bin/sup-config:23:in `<main>'
	from /Users/logan/.rvm/gems/ruby-2.2.10/bin/ruby_executable_hooks:24:in `eval'
	from /Users/logan/.rvm/gems/ruby-2.2.10/bin/ruby_executable_hooks:24:in `<main>'
```

error occurs on fresh installation. Deletion of `~/.sup` isn't help

**Comment by gauteh** [comment]
Take a look at the notmuch project. Sup is currently not maintained.

**Comment by SamyBencherif** [comment]
comment out line 120 in `/Library/Ruby/Gems/2.3.0/gems/sup-0.22.1/bin/sup-config`

<img width="864" alt="Screen Shot 2019-06-25 at 13 53 39" src="https://user-images.githubusercontent.com/10871454/60121265-ae5cb580-9750-11e9-985d-0526021fc5cd.png">

Seems to work for me. Mojave 10.14.2 (18C54)

**Comment by danc86** [comment]
It looks like this issue was fixed already in commit 7f2f6eba54ac3d122c91b783a67ab4da73bb8c2e which was unfortunately never released.

It will be included in version 0.23 which I am planning on publishing soon. In the meantime, try the latest `develop` branch of sup.

**Comment by robert-winkler** [comment]
I had the same problem on Peppermint Linux 10/ Ubuntu 18.04 LTS. Commenting L120 solved the problem.
